packages/app/src/explorer/utils.ts

```diff
@@ -26,3 +28,52 @@ export function needsNxBadge(
 
   return false;
 }
+
+function getExplorerButtonList(): HTMLButtonElement[] {
+  const explorer = document.querySelector(`#${EXPLORER_ID}`);
+  if (!explorer) {
+    return [];
+  }
+  return [...explorer.querySelectorAll('button')];
+}
+
+export function focusParent(e: { preventDefault: () => void }, path: string) {
+  const buttonList = getExplorerButtonList();
+  const parentPath = path.slice(0, path.lastIndexOf('/')) || '/';
+
+  const parentButton = buttonList.find(
+    (element) => element.getAttribute('aria-label') === parentPath
```

Comment:
This is the reason why I attached the `path` as `aria-label` in the `EntityItem`. It seemed the easiest way to retrieve the parent button.